trinity-plugins/Makefile

```diff
@@ -1,8 +1,8 @@
 
 OS := $(shell uname)
 
-CXX=g++
-CC=gcc
+CXX ?= g++
```

Comment:
I vaguely remember this causing trouble for some installations, but don't remember all the specifics.  I had to revert this change from an earlier modification.  I suppose I could push it through again now, and then we can revisit it when it crops up again.

Reply:
I see. Let's check the issue again when it will happen again in the future.